lib/src/screens/settings/desktop_settings/desktop_settings_page.dart

```diff
@@ -30,75 +33,78 @@ class _DesktopSettingsPageState extends State<DesktopSettingsPage> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Container(
-        height: MediaQuery.of(context).size.height,
-        child: Column(
-          mainAxisSize: MainAxisSize.min,
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            Padding(
-              padding: const EdgeInsets.all(24),
-              child: Text(
-                S.current.settings,
-                style: textXLarge(),
+      body: BottomSheetListener(
```

Comment:
why add the listener on the settings page? why not on the dashboard page?

Reply:
In Desktop, the Navigator pops off the route as we use separate Navigators for the routes. 

What that means is that it disposes the BottomSheet Listener that is attached to the Dashboard page, hence we need to add the listener here to handle the flow for the Desktop app.